prefetch.bs

```diff
@@ -201,7 +205,7 @@ A <dfn>cross-partition prefetch state</dfn> is a [=struct=] with the following [
 * <dfn for="cross-partition prefetch state">isolated partition key</dfn>, a [=network partition key=] whose first item is an [=opaque origin=] and which represents a separate partition in which state can be temporarily stored
 * <dfn for="cross-partition prefetch state">origins with conflicting credentials</dfn>, an [=ordered set=] of [=origins=] (initially empty)
 
-<p class="note">Prefetches which start in a different partition as their referrer (e.g., because the URL is cross-site) abort when they would return to that partition (and thus would be credentialed normally), and when an origin for which credentials would have been sent is encountered.</p>
+<p class="note">Prefetches which start in a different partition as their referrer (e.g., because the URL is cross-site) abort when they would return to that partition (and thus would be credentialed normally), and when an origin for which credentials would have been sent was encountered (unless the response specifically indicates otherwise in a [:Supports-Loading-Mode:] header field).</p>
```

Comment:
Maybe flip this for clarity? "Unless the response indicates otherwise using `Supports-Loading-Mode: uncredentialed-prefetch`, prefetches which start..."

Reply:
Rephrased. It's a little awkward because there are two related ideas captured in this single note.